purview/api/v1.py

```diff
@@ -17,3 +18,13 @@ def about():
     :return: A JSON object with version info
     """
     return jsonify({'version': version})
+
+
+@api_v1.route('/<resource>/<id>')
+def get_resource(resource, id):
+    for model in all_models:
+        if model.__name__.lower() == resource:
+            try:
+                return model.nodes.get(id_=id).serialized
+            except model.DoesNotExist:
+                return 'Requested {0} does not exist!'.format(resource)
```

Comment:
This should be JSON. How about something like:
```python
@api_v1.route('/<resource>/<id>')
def get_resource(resource, id):
    for model in all_models:
        if model.__name__.lower() == resource:
            item = model.nodes.get_or_none(id_=id)
            if item:
                return jsonify(item.serialized)
    raise NotFound('The item does not exist.')
```


Then you'd need to do this import:
```python
from werkzeug.exceptions import NotFound
```